packages/auth-providers/dbAuth/middleware/src/index.ts

```diff
@@ -18,12 +18,14 @@ export interface DbAuthMiddlewareOptions {
     req: Request | APIGatewayProxyEvent,
     context?: Context,
   ) => DbAuthResponse
+  extractRoles?: (decoded: any) => string[]
   getCurrentUser: GetCurrentUser
 }
 
 export const initDbAuthMiddleware = ({
```

Comment:
Sorry, remind me again why dbAuth is `init` and Supabase is `create`?

Reply:
I just haven't changed supabase yet :)))